DataStructures/BitArray/TestBitArray.cs

```diff
@@ -0,0 +1,377 @@
+﻿/**
+ * Author: Christian Bender
```

Comment:
Please remove the author information from this class. We maintain a high-level list of contributors in the README.md file.